test/publish.c

```diff
@@ -161,14 +162,256 @@ static void TestLoopbackAckLargeMessage(DPS_Node* node)
     DPS_DestroyPublication(pub);
 }
 
+#define HISTORY_CAP 10
```

Comment:
Have you tested out of order arrival of ACKs?

Reply:
No, but should be easy to add